full-backend-tests/src/test/java/org/graylog/plugins/views/SearchMetadataIT.java

```diff
@@ -80,6 +91,7 @@ void testMinimalRequestWithSingleParameter() {
 
     @ContainerMatrixTest
     void testRetrievingMetadataForStoredSearchWithoutParameter() {
+        LOG.error("testRetrievingMetadataForStoredSearchWithoutParameter()");
```

Comment:
Should we remove this logging before merging?

Reply:
I already did